plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -569,6 +569,17 @@ protected function handle_row_actions( $item, $column_name, $primary ) : string
 			);
 		}
 
+		/**
+		 * Filters the action links displayed for each review in the Reviews list table.
+		 *
+		 * @param string[]   $actions An array of comment actions. Default actions include:
+		 *                            'Approve', 'Unapprove', 'Edit', 'Reply', 'Spam',
+		 *                            'Delete', and 'Trash'.
+		 * @param WP_Comment $item The comment object.
+	 	*/
+		
+		$actions = apply_filters( 'comment_row_actions', array_filter( $actions ), $item );
```

Comment:
@sackadelic This will affect both regular comments and WooCommerce reviews. Should we use a dedicated filter name for this case since it pertains specifically to WooCommerce product reviews? Or do you intend to impact both WordPress comments and WooCommerce product reviews?

```suggestion
		$actions = apply_filters( 'woocommerce_product_review_row_actions', array_filter( $actions ), $item );
```

Reply:
Thanks for the clarification. I can see that this `handle_row_actions` follows the same logic as WP core's `handle_row_actions`. It makes sense to keep the comment_row_actions filter for consistency!